Packet++/header/TcpReassembly.h

```diff
@@ -299,7 +350,7 @@ class TcpReassembly
 	 * Get a list of all connections managed by this TcpReassembly instance (both connections that are open and those that are already closed)
 	 * @return A list of all connections managed. Notice this list is constant and cannot be changed by the user
 	 */

```

Comment:
Please change from "list" to "map" where the key is flowkey

Reply:
Done